pontoon/base/views.py

```diff
@@ -281,6 +281,16 @@ def entities(request):
     return _get_paginated_entities(locale, preferred_source_locale, project, form, entities)
 
 
+def _get_translation_values(query):
+    return query.values(
+        'locale__code',
+        'locale__name',
+        'locale__direction',
+        'locale__script',
+        'string',
+    )
```

Comment:
To simplify the logic on frontend, I would convert the output to:
```python
    return [{
        'code': translation.locale.code,
        'direction': translation.locale.direction,
        'locale': translation.locale.name,
        'locale_pk': translation.locale.pk,
        'script': translation.locale.script,
        'translation': translation.string,
    } for translation in query]
```

Mind the `locale_pk`, which we'll need for sorting below.